openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.mode_changed.json

```diff
@@ -0,0 +1,34 @@
+{
+   "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+   "id":"urn:uuid:921bd866-93ac-4ed1-a147-cdd1fe91e87f",
+   "type":"Enrollment",
+   "actor":{
+      "type":"Person",
+      "id":"edx.devstack.lms:18000/u/honor"
+   },
+   "action":"Used",
+   "object":{
+      "id": "http://localhost:18000/courses/course-v1:PUCIT+CS520+2018_T4/about",
+      "type": "SoftwareApplication"
+   },
+   "eventTime":"2018-10-17T13:08:41.487Z",
+   "extensions":{
+      "extra_fields":{
+         "username":"honor",
```

Comment:
`username` is unnecessary here because it has been utilized in `id` of `actor`

Reply:
The purpose of username is to save key, that's why it was placed in extra_fields